File: Loon-0.5-Alpha/Loon-Neo-GWT/src/loon/html5/gwt/emu/loon/utils/json/JsonTypes.java
package loon.utils.json;

import loon.html5.gwt.GWTJson;

class JsonTypes {
	
  public static boolean isArray(Object o) {
    return GWTJson.isObjectAnArray(o);
  }

  public static boolean isObject(Object o) {
    return GWTJson.isObjectAnObject(o);
  }
}

File: C#/Loon2MonoGame/LoonMonoGame-Lib/loon/utils/json/JsonTypes.cs
namespace loon.utils.json
{
	internal class JsonTypes
	{

		public static bool IsArray(object o)
		{
			return o is loon.Json_Array;
		}

		public static bool IsObject(object o)
		{
			return o is loon.Json_Object;
		}

	}
}
